### ISSUE TITLE
[jshow] Ads not displayed

### ISSUE THREAD

**Issue by Sandra97** [joomla/joomla-websites#839]
It seems that the Advertisements are not displayed anymore on Showcase. 
![index](https://cloud.githubusercontent.com/assets/4682328/22565415/6319c394-e95e-11e6-8d39-73ccd944356c.png)

@aDaneInSpain @sebastienlapoux Could you give a look?

Thanks

**Comment by sebastienlapoux** [comment]
Thanks @Sandra97 . Nothing changed on this side. Joe need to check if the embedded code is always working on not.
If you know the git account of Joe please ping him. I will send to him an email.
Thanks.

**Comment by zero-24** [comment]
@joejoomla maybe?

**Comment by Sandra97** [comment]
Yes!

**Comment by JoeJoomla** [comment]
Ugh, someone must have changed something in the ad server.
I'll check this shortly

**Comment by aDaneInSpain** [comment]
Strange I did not get notified about this issue. Anyway... yes. I can only assume that showcase was using the ad code for someone else and that was deleted when I did the spring cleaning on the ad server. I have created 3 new positions for the site (correctly) and emailed @sebastienlapoux directly before being aware of this issue. This should be solved quickly.

I have created a general guide on how to add ads to our sites here: https://github.com/aDaneInSpain/joomla-websites/blob/9d199513bf3f8b952332c9580ebc94ddd4b11815/documentation/guide-for-adding-banner-ads.md
